### DISCUSSION TITLE
"plex-less" version of plex_debrid in experimental branch

### DISCUSSION THREAD

**Discussion by itsToggle** [itsToggle/plex_debrid#64] [Announcements]
Hey guys,

Ive created an experimental branch for future updates. The main branch will recieve updates from the experimental branch once they are deemed stable.

The current mayor update that can be found in the experimental branch makes the plex integration optional - you can now run plex_debrid entirely through trakt. 

The update contains a new setting called "Library Service", which lets you choose how plex_debrid determines if you have already downloaded something. 

Current features for a "trakt-only" setup of plex_debrid are:

- instantly downloading newly watchlisted content from multiple trakt.tv users watchlists
- instantly downloading new content from multiple public trakt lists
- Monitoring shows from your trakt collection for newly released episodes.
- Checking your trakt collection for already "collected" content before a download is started
- Adding content to your trakt collection after a successful download, including metadata like the downloaded resolution.

Currently known bugs are:

- Shows are always collected entirely, no matter if some episodes were not downloaded.

Upcoming features are:

- Bug fixes

**Comment by darkjezza** [comment]
Is it possible to make it work with just overseerr by its self?

**Reply by itsToggle** [reply]
at the moment no, sadly not. You need to connect it to your plex server aswell. Ill see if Im gonna go through the effort of writing it into a standalone media type, but that takes a lot of time.
